cmd/kube-aws-approver/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+
+	awsapprover "github.com/coreos/kubecsr/pkg/approver/aws"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/leaderelection"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
+	"k8s.io/client-go/tools/record"
+)
+
+var (
+	kubeconfig  = flag.String("kubeconfig", "", "kubeconfig file with acces to cluster. (testing only)")
+	regionName  = flag.String("region-name", "", "When empty uses metadata service to extract.")
+	allowedASGs = flag.String("allowed-asgs", "", "Comma separated string of allowed ASGs")
+)
+
+func main() {
```

Comment:
It would be better if you could us [cobra](https://github.com/spf13/cobra) to handle all the CLI. Example [https://github.com/kubernetes-incubator/bootkube/tree/master/cmd/bootkube](https://github.com/kubernetes-incubator/bootkube/tree/master/cmd/bootkube)

Reply:
personally cobra would be better suited if there were subcommands. stdlib flags are all we need here for this binary.

Reply:
We should use the same thing everywhere. If we're going to add more commands that use cobra then this should too.